subprojects/docs/src/docs/userguide/introductionDependencyManagement.adoc

```diff
@@ -50,7 +62,7 @@ Gradle takes your dependency declarations and repository definitions and attempt
 
 * Given a required dependency, Gradle first attempts to resolve the _module_ for that dependency. Each repository is inspected in order, searching first for a _module descriptor_ file (POM or Ivy file) that indicates the presence of that module. If no module descriptor is found, Gradle will search for the presence of the primary _module artifact_ file indicating that the module exists in the repository.
```

Comment:
"module for that dependency" sounds very vague and might be unclear. Especially because _module_ is such a general term. Maybe better: "module the dependency points at".  Also we changed the behavior that we do not always look for "the presence of the primary artifact" in addition to looking for metadata and make that configurable. I am about add something about that to the repository section. So here I would not go into detail. Also I would mention our own .module metadata files as the first preview of them will be available with the next release.

Here is a suggestion for how to change this paragraph:

> Given a required dependency, Gradle attempts to resolve the dependency by searching for the _module_ the dependency points at. Each repository is inspected in order. Depending on the [type of repository](LINK-TO-SECTION) Gradle looks for metadata files describing the module (.module, .pom or ivy.xml file) or directly for artifact files.

Also further down we say "(Ivy or POM file)" . Change that to "(.module, .pom or ivy.xml file)" as well

Reply:
> "module for that dependency" sounds very vague and might be unclear. Especially because module is such a general term.

done

> Also we changed the behavior that we do not always look for "the presence of the primary artifact" in addition to looking for metadata and make that configurable. I am about add something about that to the repository section. So here I would not go into detail. Also I would mention our own .module metadata files as the first preview of them will be available with the next release.

I'll leave that up to you guys to add. You have the best knowledge on how this works.

> Also further down we say "(Ivy or POM file)" . Change that to "(.module, .pom or ivy.xml file)" as well

Same as above.

Reply:
Ok. I added the _metadata sources_-related change to #4089